index.js

```diff
@@ -1,28 +1,24 @@
 'use strict'
 
-const path = require('path')
-
-const defaultLocal = {
-  dir: '$HOME/.persist/'
-}
-
-const defaultPersisters = [ 's3', 'local' ]
+const defaultPersisters = ['s3', 'local']
 
 class Persister {
+
   /**
    * Creates persisters for saving and loading
    *
    * @param  {Object} opts Description of persisters
    * @return {Persist}     Persister
    */
+
   constructor(services) {
     this.persisters = []
 
-    services.forEach(service => {
-      if(defaultPersisters.indexOf(service.type) > -1) {
-        let persisterModule = require(`./lib/persisters/${service.type}`)
+    services.forEach((service) => {
+      if (defaultPersisters.indexOf(service.type) > -1) {
+        const PersisterModule = require(`./lib/persisters/${service.type}`)
 
-        this.persisters.push(persisterModule(service))
+        this.persisters.push(new PersisterModule(service))
       } else {
         this.persisters.push(service.implementation)
```

Comment:
I don't think I am really following. 

The intention was they just pass an object that implements a `save`/`load` and persist doesn't keep state or any info about it. It could change where `type` has to be custom if that is what you are meaning just so each obj would have to have type. I just didn't think it important for people to have to specify. Let me know your thoughts.